cmd2.py

```diff
@@ -242,23 +242,33 @@ def strip_quotes(arg):
     return arg
 
 
+def parse_quoted_string(cmdline):
```

Comment:
Good job breaking this functionality out into a function.  That reduces duplicated code.